charts/infra-apps/values.yaml

```diff
@@ -269,3 +269,25 @@ mimir:
   # -- Helm values
   # @default -- [upstream values](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/values.yaml)
   values: {}
+
+# -- [oauth2-proxy] (https://github.com/oauth2-proxy/oauth2-proxy)
+# @default -- [example](./examples/.yaml)
```

Comment:
can you add a simple example and fix this link?

Reply:
I added an example and fixed the link